ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -31,8 +31,8 @@ OnboardingPage {
     // [{keyUid:string, username:string, thumbnailImage:string, colorId:int, colorHash:var, order:int, keycardCreatedAccount:bool}]
     required property var loginAccountsModel
 
-    property bool biometricsAvailable: Qt.platform.os === Constants.mac
```

Comment:
I think we should keep it here, and make the distinction. Because even though we might have a saved credential for a given keyUid, it doesn't necessarily mean that the biometrics will work. The user might disable it globally later

Reply:
I added additional check for `biometricsAvailable` on the upper level when resolving `isBiometricsLogin`